ast/Trees.cc

```diff
@@ -811,6 +811,8 @@ string Literal::toStringWithTabs(const core::GlobalState &gs, int tabs) const {
     string res;
     typecase(
         this->value, [&](const core::LiteralType &l) { res = l.showValue(gs); },
```

Comment:
Can we replace `LiteralType` with something like `LiteralNamedType` or something?

I don't want to accidentally think that `LiteralType` still represents all literals and type out `LiteralType` out of habit.